shared/common-adapters/platform-icon.js

```diff
@@ -8,51 +8,75 @@ type Props = {
   platform: PlatformsExpandedType,
   size: 48,
   overlay: IconType,
-  overlayColor: string,
+  overlayColor?: string,
   style?: Object,
 }
 
-const supportedPlatformsAndSizes: {[key: PlatformsExpandedType]: ?{[key: string]: ?IconType}} = {
-  'coinbase': {
-    '48': 'icon-coinbase-logo-48',
-  },
-  'twitter': {
-    '48': 'icon-twitter-logo-48',
-  },
-  'github': {
-    '48': 'icon-github-logo-48',
-  },
-  'reddit': {
-    '48': 'icon-reddit-logo-48',
-  },
-  'hackernews': {
-    '48': 'icon-hacker-news-logo-48',
-  },
-  'dns': {
-    '48': 'icon-website-48',
-  },
-  'http': {
-    '48': 'icon-website-48',
-  },
-  'https': {
-    '48': 'icon-website-48',
-  },
-  'btc': {
-    '48': 'icon-bitcoin-logo-48',
-  },
+type IconSpec = {
+  icon: IconType,
+  offsetBottom: number,
+  offsetRight: number,
+}
+
+const standardOffsets = {
+  offsetBottom: -2,
+  offsetRight: -5,
+}
+
+const getSpecForPlatform = (platform: PlatformsExpandedType): {[size: string]: IconSpec} => {
+  const specs = {
+    'coinbase': {
+      '48': {icon: 'icon-coinbase-logo-48', ...standardOffsets},
+    },
+    'twitter': {
+      '48': {icon: 'icon-twitter-logo-48', ...standardOffsets},
+    },
+    'github': {
+      '48': {icon: 'icon-github-logo-48', ...standardOffsets},
+    },
+    'reddit': {
+      '48': {icon: 'icon-reddit-logo-48', ...standardOffsets},
+    },
+    'hackernews': {
+      '48': {icon: 'icon-hacker-news-logo-48', ...standardOffsets},
+    },
+    'dns': {
+      '48': {icon: 'icon-website-48', ...standardOffsets},
+    },
+    'http': {
+      '48': {icon: 'icon-website-48', ...standardOffsets},
+    },
+    'https': {
+      '48': {icon: 'icon-website-48', ...standardOffsets},
+    },
+    'dnsOrGenericWebSite': {
+      '48': {icon: 'icon-website-48', ...standardOffsets},
+    },
+    'rooter': {
+      '48': {icon: 'icon-website-48', ...standardOffsets},
+    },
+    'btc': {
+      '48': {icon: 'icon-bitcoin-logo-48', ...standardOffsets},
+    },
+    'pgp': {
+      '48': {icon: 'icon-pgp-key-48', offsetBottom: -2, offsetRight: 4},
+    },
+  }
+  return specs[platform]
 }
 
 const Render = ({platform, overlay, overlayColor, size, style}: Props) => {
```

Comment:
I agree, but it is already wired up and the outside world doesn't really have to deal with it much. It would be a pain to re-add it if we ended up needing it later down the road.